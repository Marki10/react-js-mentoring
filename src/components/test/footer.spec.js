import React, { Component } from 'react';
import Footer from '../footer';
const ReactTestRenderer = require('react-test-renderer');

describe('Footer', () => {
    it('Render footer', () => {
        const component = ReactTestRenderer.create(
            <Footer />
        );
        const testTemp = component.toJSON();
        expect(testTemp).toMatchSnapshot();
    })
});