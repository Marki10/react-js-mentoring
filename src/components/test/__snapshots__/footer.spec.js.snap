exports[`Footer Render footer 1`] = `
<footer
  className="container footer">
  netflixroulette
</footer>
`;
